app/views/moments/_form.html.erb

```diff
@@ -102,6 +102,22 @@
 
       <%= render :partial => '/shared/character_count', locals: { data: 'moment_fix' } %>
     </div>
+    <div class="field">
```

Comment:
Looks like we could move this into partial so that strategies can use it too instead of having this template duplicated twice.